### PR TITLE
Update README with link to Learn and image size details

### LINKED ISSUE #992: Improve the README for the image requirements
The README says images must be a certain width, but the tests are way more strict that that. The README should be updated to reflect the more specific requirements.

I'll do a PR for this...

### PR DESCRIPTION
**Pull request by askpatrickw**
Fixes #992

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+following Adafruit Learn guide: [How to add a New Board to the circuitpython.org website](https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website)
 
 **To add a new board to the site:**
 
 1. Duplicate `template.md` to `_board/<board id>.md`.
 2. Edit `_board/<board id>.md` according to the template's instructions.
-3. Provide 3 images. An original high-quality image. A smaller image (300 px width),
-and a larger image (700 px width) in each respective directory (assets/images/boards/{small large original})
-and process them in something like https://squoosh.app/ to reduce file size. If
+3. In your `_board/<board id>.md` you will specifigy a `board_image`. Create 3
```
Comment (makermelissa) [inline README.md:10]: Perhaps you mean "specify" here?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+versions of this file, in the following sizes and folder locations:
+
+|Size|File|Dimensions (px)|
+|--|--|--|
+|Original|assets/images/orignal/{board_image}|900px+ width and 4:3 ratio width|
```
Comment (makermelissa) [inline README.md:15]: Some of the images are 13:10 which is fine. Min widths are 292px for small and 590px for large, but these are absolute minimums and could be a little bigger like you suggested. Original Image minimum size is the same as the large, and 900px or larger is sometimes difficult or impossible to find.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
+|||example: 900 x 675|
+|Large|assets/images/large/{board_image}|800 x 600|
+|Small|assets/images/small/{board_image}|300 x 225|
+
+> For more information on preferred images see
+> [Preparing the Images](https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/preparing-the-images)
+> in the Adafruit Learn guide
+
+
+4. Process the in something like https://squoosh.app/ to reduce file size. If
```
Comment (makermelissa) [inline README.md:25]: No need to compress the images. That is done automatically in GitHub Actions.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Thanks for updating. See above for specific items to fix.

File: README.md (hunk 1 above)
Comment (askpatrickw) [reply README.md]: ty

File: README.md (hunk 2 above)
Comment (askpatrickw) [reply README.md:15]: I added a note about seeing the docs for 13:10.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Thanks for making those changes. One last little thing...

File: README.md [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
+|Original|assets/images/orignal/{board_image}|900px+ width and 4:3 ratio width|
+|||example: 900 x 675|
+|Large|assets/images/large/{board_image}|800 x 600|
+|Small|assets/images/small/{board_image}|300 x 225|
+
+> For more information on preferred images or if you prefer or must use 13:10 see
+> [Preparing the Images](https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/preparing-the-images)
+> in the Adafruit Learn guide
+
+5. Create a pull request with the file changes.
```
[the remark this answers is not in the document]
Comment (askpatrickw) [reply README.md:24]:
```suggestion
4. Create a pull request with the file changes.
```

**Review by makermelissa — APPROVED** [review]
Thanks

**Pull request merged**